hana/primary_available.sls

```diff
@@ -0,0 +1,17 @@
+{%- from "hana/map.jinja" import hana with context -%}
+{% set host = grains['host'] %}
+
+{% for node in hana.nodes %}
+{% if node.host == host and node.secondary is defined %}
+netcat-openbsd:
```

Comment:
actually I think I have a better way to do this in hacluster-formula where it looks for Hawk to be running, no need for netcat: https://github.com/SUSE/habootstrap-formula/blob/master/cluster/join.sls#L3

Reply:
Agree! The other way looks better, less dependencies. I will make the update and test it.
Thanks @krig

Reply:
ah but couldn't you use the 7630 port here too?

Reply:
aha no you need to check if HANA is running of course, sorry!

Reply:
Exactly, we need to check is System replication of HANA is enabled. There are many ports for that, but none of them uses http protocol

Reply:
I see yeah. Hm, seems salt doesn't come with anything better, at least at a glance..